app/src/main/java/com/appdev/eateryblueandroid/ui/components/login/TitleSection.kt

```diff
@@ -13,17 +11,22 @@ import androidx.compose.ui.unit.dp
 import com.appdev.eateryblueandroid.R
 import com.appdev.eateryblueandroid.ui.components.core.Text
 import com.appdev.eateryblueandroid.ui.components.core.TextStyle
+import com.appdev.eateryblueandroid.ui.viewmodels.BottomSheetViewModel
 
 @Composable
-fun TitleSection() {
+fun TitleSection(transitionSettings: () -> Unit, isLoggingIn: Boolean) {
     Row(
         modifier = Modifier.fillMaxWidth(),
         horizontalArrangement = Arrangement.End
     ) {
         Icon(
             painter = painterResource(id = R.drawable.ic_settings),
             contentDescription = null,
-            tint = colorResource(id = R.color.black)
+            tint = colorResource(id = R.color.black),
+            modifier = Modifier.then(
+                if(isLoggingIn) Modifier
```

Comment:
Is this the best way to do a if clickable modifier? Would an alternative be Modifier.clickable{ if (!isLoggingIn) transitionSettings() }?